src/pages/history.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import PageTemplate from "@/components/templates/PageTemplate";
+import History from "@/components/organisms/History";
+
+export default function Index() {
```

Comment:
Is the function name need to be `Index`? Is it possible to rename it to `History` same as the filename?

Reply:
Updated!

Reply:
Updated to HistoryIndex as History clashes with the component we are importing into the file.